solver/src/main.rs

```diff
@@ -200,12 +200,12 @@ async fn main() {
 
     let pool_aggregator = PoolAggregator::from_providers(&pair_providers, &web3).await;
 
-    // TODO - use Filtered-Cached PoolFetchers here too.
+    // TODO: use caching pool fetcher
     let price_estimator = Arc::new(BaselinePriceEstimator::new(
         Box::new(pool_aggregator),
         gas_price_estimator.clone(),
         base_tokens.clone(),
-        Arc::new(ListBasedDetector::deny_list(args.shared.unsupported_tokens)),
```

Comment:
Do we use `unsupported_tokens` in other places in the driver? If not, we should probably move it from the `shared::Options` to be `orderbook` specific.

Reply:
It's not used so I will add that to this PR.

Reply:
Maybe also add a comment why it's fine to not filter additional tokens (the orderbook does the filtering for us)?